index.bs

```diff
@@ -1866,6 +1866,13 @@ usage.
       not be found.
     </td>
   </tr>
+  <tr>
+    <td>{{NotReadableError}}</td>
+    <td>
+      The operation failed because the file(s) containing the requested
```

Comment:
Maybe "underlying storage" instead of "file(s)" since we don't require that it actually bottoms out in files?

Reply:
Done